### ISSUE TITLE
Update one: Citigroup Inc Corp espionage

### ISSUE THREAD

**Issue by ghost** [XmattwoodsX/Letter-to-NYT#1]
(808) 209-1429

Hi Jake Tapper

TLDR: Citigroup Inc just launched a denial of service attack to XignalX Inc: a struggling startup owned by former US Navy SEALs in the SF Bay area.

Please see my former letter to the NYT: https://documentcloud.adobe.com/link/review?uri=urn:aaid:scds:US:8016c2b0-e71f-4c2c-995d-b67174459387#pageNum=1

This is the second attack by Citigroup Inc on XignalX Inc... Both in broad daylight. The first time, I was in Oakland, CA authoring the NYT letter. They hacked, on my then iphone, the Citigroup iOS app, web app, the call center, and, indeed, in person at a branch in Oakland. During the previous episode, the teller at the Citi branch (4101 Piedmont Ave, Oakland, CA 94611) told me the amount of my balance, then immediately got locked out of control from their system. They WOULD NOT GIVE ME MY MONEY... DESPITE ME PROVIDING PROPER ID AT THE SCENE. THEY WOULD NOT RELEASE MY FUNDS. This is a direct connection to the Nazi network that seems to be operating with impunity in America right now, as well as Trump and his network.

This recent episode is identical. They are doing a repeat of the first attack, but this time I have no ID, a prepaid cell phone and the debit card. They are lying... Just as wells Fargo did back in 2019, when I quit Alphabet Inc. They are doing everything in their power to hack my finances to prevent me from speaking out. It is oppression, by the Nazis, wrt my family and the situation I described in the earlier letter. I advised Citigroup Inc over the phone of impending legal action and the Nazi operative told me that I was warned to shut up. Then they informed me that they would not unblock the card and went dead silent on the other end... As if to give me one final FUCK YOU right to my face. As if they are above the law. I want Citigroup to wind down. They THINK they are "too big to fail"... I BEG TO DISAGREE. 

I sit at the big island Hawaii now, under relentless attack from industry and State. They won't stop harassing me for no reason. I must conclude that they are part of the Nazi movement and are cooperating with Alphabet Inc to disrupt the activities of XignalEx, Inc. Barbaric and cruel for a former SEAL owned startup. 

I have been locked out of all my normal email accounts because my Titan key was stolen (along with my last meager possesions like a tent and backpack and Last Will and Testament/Advance Directive. They obviously think they are going to murder me before anyone finds out. The police even joke about that point right in front of my face... Murdering/torturing me. Someone please call these assholes out for Trump's fascist control over the US government/military industrial complex. They have LITERALLY LOCKED ME OUT OF MY CHECKING ACCOUNT ILLEGALLY. THEY ARE NAZIS. 

FULL STOP.
